local/php/envs.go

```diff
@@ -29,20 +29,37 @@ import (
 	"github.com/symfony-cli/symfony-cli/envs"
 )
 
-func (p *Server) generateEnv(req *http.Request) map[string]string {
-	scriptName := p.passthru
-	https := ""
-	if req.TLS != nil {
-		https = "On"
-	}
+func (p *Server) resolveIndexFile(pathInfo string) (string, string) {
 
-	pathInfo := req.URL.Path
 	if pos := strings.Index(strings.ToLower(pathInfo), ".php"); pos != -1 {
 		file := pathInfo[:pos+4]
 		if _, err := os.Stat(filepath.Join(p.documentRoot, file)); err == nil {
-			scriptName = file
-			pathInfo = pathInfo[pos+4:]
+			return file, pathInfo[pos+4:]
 		}
+
+	}
+
+	if len(pathInfo) > 1 {
+		paths := strings.Split(strings.Trim(pathInfo, "/"), "/")
+		for n := len(paths); n > 0; n-- {
+			indexDir := filepath.Join(paths[:n]...)
+			file := filepath.Join(indexDir, p.passthru)
+			if _, err := os.Stat(filepath.Join(p.documentRoot, file)); err == nil {
+				return "/" + file, pathInfo[strings.Index(pathInfo, indexDir)+len(indexDir):]
```

Comment:
as we trim/clean the pathinfo, I see a problem here where the CLI would change the pathinfo and indexDir, resulting in a wrong returned pathInfo as we use indexes and length from different cleaned strings.

For example, `"/subdirectory///subdirectory//foo/"` would cause a weird `"tory//foo/"` pathinfo returned.

Reply:
Using the `LastIndex` and length of `paths[n-1]` seems logical to me
but then some tests fails

Reply:
ok found it, actually it's because the loop considers empty elements from `paths`
an additional if allows to fix this.
@maciekpaprocki I have a patch with the suggested changes here and the additional test case: see https://github.com/maciekpaprocki/symfony-cli/pull/1